CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This release has deprecations. Please read [DEPRECATION] entries and consult
 the [upgrade guide](https://github.com/grafana/agent/blob/main/docs/upgrade-guide/_index.md)
 for detailed information.
 
+- [FEATURE] Added argument support to GrafanaAgent Custom Resource Definition.
```

Comment:
Can you rebase on top of current main and move this to the unreleased section?

Reply:
done